<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoldierDialog</class>
 <widget class="QDialog" name="SoldierDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>עריכת חייל</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>מ.א</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pnEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>שם החייל</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="soldierNameEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>מחלקה</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="platoonCombo">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>מפל&quot;ג</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>מספר טלפון</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="telephoneEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>תפקיד</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="RolesWidget" name="rolesWidget" native="true">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>הערות</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="commentEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>היעדרויות</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="absencesView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addAbsenceButton">
           <property name="text">
            <string>הוסף היעדרות</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeAbsenceButton">
           <property name="text">
            <string>הסר היעדרות</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>מאפיינים</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="manualAssignCheck">
         <property name="text">
          <string>שיבוץ ידני בלבד</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="noPhysicalCheck">
         <property name="text">
          <string>לא משימות פיזיות</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>עמדות</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="allPositionsRadio">
          <property name="text">
           <string>הכל</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="whiteListRadio">
          <property name="text">
           <string>רשימה לבנה</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="blackListRadio">
          <property name="text">
           <string>רשימה שחורה</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="positionsList">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RolesWidget</class>
   <extends>QWidget</extends>
   <header>Ui.Promoted.RolesWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pnEdit</tabstop>
  <tabstop>soldierNameEdit</tabstop>
  <tabstop>platoonCombo</tabstop>
  <tabstop>telephoneEdit</tabstop>
  <tabstop>rolesWidget</tabstop>
  <tabstop>commentEdit</tabstop>
  <tabstop>addAbsenceButton</tabstop>
  <tabstop>removeAbsenceButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SoldierDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>655</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SoldierDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>655</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addAbsenceButton</sender>
   <signal>clicked()</signal>
   <receiver>SoldierDialog</receiver>
   <slot>addAbsence()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeAbsenceButton</sender>
   <signal>clicked()</signal>
   <receiver>SoldierDialog</receiver>
   <slot>removeAbsence()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allPositionsRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>positionsList</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addAbsence()</slot>
  <slot>removeAbsence()</slot>
 </slots>
</ui>
